content/docs/tutorials/certificate-defaults/README.md

```diff
@@ -0,0 +1,473 @@
+    kubectl patch cpol 0-mutate-certificate-defaults --type=merge -p '{"spec":{"schemaValidation": false}}'
+    ```
+
+    > **NOTE**: Please read about the validation setting before using elsewhere. We have disabled it here because we have two policies both affecting `Certificates`. Cert-manager will validate the resolved resource with the `validatingwebhookconfigurations.admissionregistration.k8s.io`, after all mutations are applied. You could avoid this completely by using one policy with all required mutation rules.
+
+1. Take a look at another new `Certificate` we have for validating that all our policies will now take effect. This time called "test-minimal":
+
+    ```yaml file=./yamls/cert-test-minimal.yaml
+    ```
+
+1. Dry run apply and `diff` to validate all our defaults have applied to this minimal `Certificate`:
+
+    ```shell
+    kubectl apply -f yamls/cert-test-minimal.yaml --dry-run=server -o yaml | diff -uZ yamls/cert-test-minimal.yaml -
+    ```
+
+    An example succesful output:
+
+    ```yaml
+    --- yamls/cert-test-minimal.yaml	2023-09-07 16:58:44.979597651 +0100
+    +++ -	2023-09-08 11:05:44.122807372 +0100
+    @@ -1,8 +1,31 @@
+    apiVersion: cert-manager.io/v1
+    kind: Certificate
+    metadata:
+    +  annotations:
+    +    kubectl.kubernetes.io/last-applied-configuration: |
+    +      {"apiVersion":"cert-manager.io/v1","kind":"Certificate","metadata":{"annotations":{},"name":"test-minimal","namespace":"default"},"spec":{"dnsNames":["example.com"]}}
+    +    policies.kyverno.io/last-applied-patches: |
+    +      set-default-issuer-ref.1-mutate-certificate-required.kyverno.io: added /spec/issuerRef
+    +      set-default-secret-name.1-mutate-certificate-required.kyverno.io: added /spec/secretName
+    +      set-privateKey-details.0-mutate-certificate-defaults.kyverno.io: added /spec/privateKey/size
+    +      set-privateKey-rotationPolicy.0-mutate-certificate-defaults.kyverno.io: added /spec/privateKey
+    +      set-revisionHistoryLimit.0-mutate-certificate-defaults.kyverno.io: added /spec/revisionHistoryLimit
+    +  creationTimestamp: "2023-09-08T10:05:44Z"
+    +  generation: 1
+      name: test-minimal
+      namespace: default
+    +  uid: 269a7194-8abe-4ed3-af7b-f21c3c2ee32e
+    spec:
+      dnsNames:
+      - example.com
+    +  issuerRef:
+    +    group: cert-manager.io
+    +    kind: ClusterIssuer
+    +    name: our-corp-issuer
+    +  privateKey:
+    +    algorithm: ECDSA
+    +    encoding: PKCS1
+    +    rotationPolicy: Always
+    +    size: 521
+    +  revisionHistoryLimit: 2
+    +  secretName: test-minimal-cert
+    ```
+
+    See how both Kyverno `ClusterPolicies` have been applied and resulted in all the other default values being inserted for us!
+
+1. To be absolutely sure we have not enforced any settings, let us try and set every value of the `Certificate` explicitly, that we have a default rule for. We will use a final `Certificate` named "test-revision-override":
```

Comment:
This is me being terrible at my only fluent language. Thank you for pointing these out.
I hope I have addressed most of these now. I've tried to avoid using the term if there is a better alternative.